### ISSUE TITLE
Launcher crashes & reboots when type muitibyte characters

### ISSUE THREAD

**Issue by kaz6120** [pop-os/shell#1038]
**(1) Issue/Bug Description:**

Launcher doesn’t accept multibyte characters. When type mutibyte characters in launcher text field,  pop shell crashes & reboots. 

**(2) Steps to reproduce (if you know):**

1) Enable mutibyte input method. (ex. Japanese Mozc in my case)
2) Type something in mutibyte characters by using multibyte input method. 

**(3) Expected behavior:**

Option 1) Automatically switches to English input method
Option 2) Accept mutibyte characters in Launcher text field. 

**(4) Distribution (run `cat /etc/os-release`):**
 
 Pop!_OS 21.04 (COSMIC) latest version of 21.04

**(5) Gnome Shell version:**

3.38.4

**(6) Pop Shell version (run `apt policy pop-shell` or provide the latest commit if building locally):**

```
pop-shell:
  Installed: 1.1.0~1624922027~21.04~3f417b4
  Candidate: 1.1.0~1624922027~21.04~3f417b4
  Version table:
 *** 1.1.0~1624922027~21.04~3f417b4 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu hirsute/main amd64 Packages
       1001 http://ppa.launchpad.net/system76/pop/ubuntu hirsute/main i386 Packages
        100 /var/lib/dpkg/status
```

**(7) Where was Pop Shell installed from:**

Pop!_OS 21.04 update from Beta version.

**(8) Monitor Setup (2 x 1080p, 4K, Primary(Horizontal), Secondary(Vertical), etc):**

2560 x 1440 

**(9) Other Installed/Enabled Extensions:**

**(10) Other Notes:**

**Comment by jacobgkau** [comment]
Probably the same issue as #967.

**Comment by kaz6120** [comment]
Thanks a lot for letting me know the issue #967 
I read the post and yes, exactly the same issue. 
hit "super", type multibyte text with multibyte input method (such as Japanese / chinese), it crashes.

**Comment by jacobgkau** [comment]
Thank you for confirming. I definitely want this to get addressed, so I've renamed #967 so it's not specific to Chinese, and I'm going to close this one as a duplicate. It's easier to see the full impact of an issue when the oldest one is used for bug tracking.
